action.yml

```
@@ -46,11 +46,35 @@ runs:
     - name: Download hadolint and make it available in path
       shell: bash
       run: |
-        echo "::debug::Downloading Hadolint ${{ inputs.version }}"
-        mkdir ${{ github.action_path }}/bin
-        curl -L -s -o ${{ github.action_path }}/bin/hadolint \
-          "https://github.com/hadolint/hadolint/releases/download/v${{ inputs.version }}/hadolint-Linux-x86_64"
-        chmod +x ${{ github.action_path }}/bin/hadolint
+        download_hadolint=false
+        # Check if hadolint exists
+        if [ ! -f "${{ github.action_path }}/bin/hadolint" ]; then
```

Lets check if its executable
```suggestion
        if [ ! -x "${{ github.action_path }}/bin/hadolint" ]; then
```